stock_picking_show_return/models/stock_picking.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2014-2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    returned_ids = fields.Many2many(
```

Comment:
We should use table name (Eg. `relation='returned_picking_relation'`) explicitly to prevent sharing same table name (In case one have m2m with same object)
Related PR on Github: https://github.com/odoo/odoo/pull/29376 and https://github.com/odoo/odoo/pull/29323

Reply:
@Swapnesh-SerpentCS  As it is non-store field it will not create any database table